ui/raidboss/data/05-shb/eureka/zadnor.js

```diff
@@ -81,30 +133,1155 @@ export default {
+        text: {
+          // TODO: during spiral scourge could be "get under middle/outer light orb"?
+          en: 'Get Under Light Orb',
+          de: 'Unter einem Lichtorb stellen',
+          fr: 'Allez sous un Orbe lumineux',
+          ja: '白玉へ',
+          cn: '靠近白球',
+          ko: '하얀 구슬 안으로',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Saunion Tooth and Talon',
+      netRegex: NetRegexes.startsUsing({ source: 'Dawon The Younger', id: '5DD4' }),
+      condition: tankBusterOnParty(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Zadnor Saunion Swooping Frenzy',
+      netRegex: NetRegexes.startsUsing({ source: 'Dawon The Younger', id: '5DD0', capture: false }),
+      infoText: (data, _matches, output) => {
+        // Every other Swooping Frenzy is followed by a Frigid Pulse, starting with the first.
+        data.saunionSwoopingCount = (data.saunionSwoopingCount || 0) + 1;
+        if (data.saunionSwoopingCount % 2)
+          return output.text();
+      },
+      outputStrings: {
+        text: {
+          en: 'Follow Dawon',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Diablo Advanced Death Ray',
+      netRegex: NetRegexes.headMarker({ id: '00E6' }),
+      condition: (data) => data.ce === 'dalriadaDiablo',
+      // TODO: this is maybe worth promoting to responses?
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          tankLaserOnYou: {
+            en: 'Tank Laser on YOU',
+          },
+          avoidTankLaser: {
+            en: 'Avoid Tank Laser',
+          },
+        };
+
+        if (data.me === matches.target)
+          return { alarmText: output.tankLaserOnYou() };
+        return { infoText: output.avoidTankLaser() };
+      },
+    },
+    {
+      id: 'Zadnor Diablo Aetheric Explosion',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CC6', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Diablo Ultimate Psuedoterror',
+      // This is triggered on Diabolic Gate with a delay, so it gives an extra +4 seconds.
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5C9F', capture: false }),
+      delaySeconds: 35,
+      response: Responses.getUnder(),
+    },
+    {
+      id: 'Zadnor Diablo Advanced Death IV',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CAF', capture: false }),
+      // Circles appear at the end of the cast.
+      delaySeconds: 4,
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Avoid Growing Circles',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Diablo Advanced Death IV Followup',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diablo Armament', id: '5CAF', capture: false }),
+      delaySeconds: 12,
+      // TODO: or "Avoid Growing Circles (again lol)"?
```

Comment:
Could bump the severity on the second one?

Reply:
I wanted this to be alert -> info just to not be overwhelming.